### ISSUE TITLE
Add link to "Edit profile" page from "Profile"

### ISSUE THREAD

**Issue by ck3g** [ck3g/homebugh#28]
The profile page should have a link to Edit profile page.

⚠️ Blocked by https://github.com/ck3g/homebugh/issues/26

* Profile page URL `/users/profile` (TBD in https://github.com/ck3g/homebugh/issues/26)
* Edit profile page URL `/users/edit`

**Comment by chethanagopinath** [comment]
Hi @ck3g ! Would love to pick this up if possible.

**Comment by ck3g** [comment]
Hey @chethanagopinath that's great! But keep in mind the profile page should be implemented with this [issue](https://github.com/ck3g/homebugh/issues/26) first.

**Comment by chethanagopinath** [comment]
@ck3g Okay, so these are like two separate issues? I'm looking to participate in HacktoberFest and I found that issue first due to the label :)

**Comment by ck3g** [comment]
Yeah, there are several issues labeled with "Hacktoberfest". I'm going to create a bunch more.

**Comment by chethanagopinath** [comment]
Awesome! @ck3g I think I'll try picking another issue within your repo, I'm looking for beginner issues which I can work on.
